apps/fluent-tester/src/TestComponents/ColorTokens/ColorTokenTest.tsx

```diff
@@ -29,52 +30,59 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     marginVertical: 5,
   },
-  pickerContainer: {
-    flexDirection: 'row',
-    justifyContent: 'space-evenly',
-  },
   stackStyle: {
     borderWidth: 2,
     padding: 12,
     margin: 8,
   },
 });
 
-const getSwatchColorStyle = (name: string, color: ColorValue): ViewStyle => {
-  styles[name] = styles[name] || { backgroundColor: color };
-  return styles[name];
+const getSwatchColorStyle = (colorName: string, colorValue: ColorValue): ViewStyle => {
+  styles[colorName] = styles[colorName] || { backgroundColor: colorValue };
+  return styles[colorName];
 };
 
-type ColorTokenProps = { color: ColorValue; name: string };
+type ColorTokenProps = { colorValue: ColorValue; colorName: string };
 const ColorToken: React.FunctionComponent<ColorTokenProps> = (p: ColorTokenProps) => {
+  if (p.colorValue === undefined) {
+    console.log('Color token ' + p.colorName + ' is undefined');
+  }
```

Comment:
Should this be a yellow box warning?

Reply:
Which one is yellow box? I had this as console.warn at first, wanted to try this in case that fixed the e2e tests but since it didn't will probably go back to console.warn

Reply:
I think console.warn()

Reply:
Changed it back to console.warn